### PR TITLE
Dynamic voting redesign

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Design and textual changes for `VotesResults` component.

# Changes

- update descriptions
- position of cast votes
- hide cast votes for default proposals

# Tests

- extend VotesResults

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary

# Screenshots

## Before

image

## After

Screenshot 2023-12-11 at 14 25 02

### CODE REVIEW

File: frontend/src/lib/components/proposal-detail/VotesResults.svelte [hunk 1]
```diff
@@ -42,20 +42,17 @@
...
+  $: castVotesYes = yesNoSum > 0 ? (yes / yesNoSum) * 100 : 0;
+  let castVotesNo: number;
+  $: castVotesNo = yesNoSum > 0 ? (no / yesNoSum) * 100 : 0;
 
   let showExpirationDate: boolean = true;
   $: showExpirationDate =
     nonNullish(deadlineTimestampSeconds) &&
     deadlineTimestampSeconds > BigInt(nowInSeconds());
 
+  // TODO(max): refactor to "isCritical", isSuperMajority -> isCriticalProposal
```
Comment (lmuntaner) [inline frontend/src/lib/components/proposal-detail/VotesResults.svelte:55]: What does this todo mean?

File: frontend/src/tests/page-objects/VotesResults.page-object.ts [hunk 2]
```diff
@@ -51,6 +51,16 @@ export class VotesResultPo extends BasePageObject {
     );
   }
 
+  /** @returns formatted text */
```
Comment (lmuntaner) [inline frontend/src/tests/page-objects/VotesResults.page-object.ts:54]: Why this comment?

**Review by lmuntaner — APPROVED** [review]
LGTM!

File: frontend/src/lib/components/proposal-detail/VotesResults.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/proposal-detail/VotesResults.svelte:55]: Thanks, refactored.

File: frontend/src/tests/page-objects/VotesResults.page-object.ts (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/tests/page-objects/VotesResults.page-object.ts]: Removed. This is relict from other place when AFAIR it was an html an we did care about it.

**Pull request merged**